src/app/user-page/user-page.component.html

```diff
@@ -0,0 +1,132 @@
+                  <span>{{ gitHubProfile.username }}</span>
+                </a>
+              </span>
+            </mat-card-subtitle>
+          </mat-card-header>
+          <span>
+            <img [src]="gitHubProfile.avatarURL" alt="GitHub avatar" />
+          </span>
+        </div>
+        <div *ngIf="!user?.orcid && !googleProfile" fxLayout="column" fxLayoutAlign="end" class="mt-3">
+          <hr />
+          <span fxLayout fxLayoutAlign="end end">
+            <a href="https://github.com/{{ gitHubProfile?.username }}" target="_blank">
+              <button mat-button class="accent-1-dark-btn small-mat-btn-skin small-btn-structure mt-2">
+                <mat-icon class="mat-icon-sm">open_in_new</mat-icon>
+                View
+              </button>
+            </a>
+          </span>
+        </div>
+      </mat-card>
+      <!-- ORCID -->
+      <mat-card *ngIf="user?.orcid" class="h-100">
+        <mat-card-header>
+          <div mat-card-avatar matBadge="✓" class="mr-2">
+            <img src="../../assets/images/account-logos/orcid.svg" alt="ORCID logo" class="w-100" />
+          </div>
+          <mat-card-title class="normal-line-height">ORCID Account</mat-card-title>
+          <mat-card-subtitle class="accent-1-dark normal-line-height mt-2 mb-0">
+            <a href="https://orcid.org/{{ user.orcid }}" target="_blank">
+              <span class="weight-bold">https://orcid.org/{{ user?.orcid }}</span>
+            </a>
+          </mat-card-subtitle>
+        </mat-card-header>
+      </mat-card>
+      <!-- Google -->
+      <mat-card *ngIf="googleProfile" fxLayout="column" fxLayoutAlign="space-between" class="h-100">
+        <div fxLayout fxLayoutAlign="space-between">
+          <mat-card-header>
+            <div mat-card-avatar matBadge="✓" class="mr-2">
+              <img src="../../assets/images/account-logos/google.svg" alt="Google logo" class="w-100" />
+            </div>
+            <mat-card-title class="normal-line-height">Google Account</mat-card-title>
+            <mat-card-subtitle class="accent-1-dark normal-line-height weight-bold mt-2 mb-0">
+              <span>Account Name: {{ googleProfile.name }}</span>
+            </mat-card-subtitle>
+          </mat-card-header>
+          <span>
+            <img [src]="googleProfile.avatarURL" alt="Google avatar" />
+          </span>
+        </div>
+        <div *ngIf="!user?.orcid && !gitHubProfile" fxLayout="column" fxLayoutAlign="end">
+          <hr class="mb-5" />
+        </div>
+      </mat-card>
+    </div>
+  </div>
+
+  <mat-tab-group class="pb-5 purple-tab-group">
+    <mat-tab>
+      <ng-template mat-tab-label>
+        <div fxLayoutGap="0.5rem">
+          <mat-icon>restore</mat-icon>
+          <span>Activity</span>
+        </div>
+      </ng-template>
+      <div class="p-2">
+        <app-recent-events></app-recent-events>
```

Comment:
Can you add a comment about this being a placeholder until the endpoint needed is implemented (your PR description mentioned this)